shared/src/components/Product/Theta/YieldCard.tsx

```diff
@@ -332,30 +338,61 @@ const YieldCard: React.FC<YieldCardProps> = ({
       !isPracticallyZero(vaultBalanceInAsset, decimals)
     ) {
       return (
-        <div className="d-flex w-100 justify-content-center">
-          <SecondaryText fontSize={12} color={colors.primaryText}>
-            Funds ready for migration to V2
-          </SecondaryText>
-        </div>
+        <ModalContentExtra style={{ paddingTop: 14 + 16, paddingBottom: 14 }}>
+          <div className="d-flex w-100 justify-content-center">
+            <SecondaryText fontSize={12} color={colors.primaryText}>
+              Funds ready for migration to V2
```

Comment:
I know this is unrelated but in the future, please add text to the `translation.json` so we can work down the tech debt